lib/teiserver/account/reports/retention_report.ex

```diff
@@ -70,7 +70,7 @@ defmodule Teiserver.Account.RetentionReport do
             end
           end)
 
-        if last_login != nil and last_played != nil do
```

Comment:
Why this?

Reply:
I’m following the compiler. And `Timex.from_unix` raises an error if it fails, it never returns nil. The newer compiler is now able to spot that kind of things.